### PR TITLE
357 add action.get

### PR DESCRIPTION
**Pull request by garrettH3S**

### CODE REVIEW

File: ckanext/bcgov/logic/get.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    if pagination_dict:
+        pagination_dict, errors = _validate(
+            data_dict, logic.schema.default_pagination_schema(), context)
+        if errors:
+            raise ValidationError(errors)
+    sort = data_dict.get('sort') or 'title'
+    q = data_dict.get('q')
+
+    try:
+        all_fields = asbool(data_dict.get('all_fields', None))
```
[the remark this answers is not in the document]
Comment (garrettH3S) [reply ckanext/bcgov/logic/get.py]: updated.

File: ckanext/bcgov/logic/get.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+_select = sqlalchemy.sql.select
+_aliased = sqlalchemy.orm.aliased
+_or_ = sqlalchemy.or_
+_and_ = sqlalchemy.and_
+_func = sqlalchemy.func
+_desc = sqlalchemy.desc
+_case = sqlalchemy.case
+_text = sqlalchemy.text
+
+def _group_or_org_list(context, data_dict, is_org=False):
```
[the remark this answers is not in the document]
Comment (garrettH3S) [reply ckanext/bcgov/logic/get.py]: updated.

File: ckanext/bcgov/logic/get.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+import ckan.lib.jobs as jobs
+import ckan.lib.navl.dictization_functions
+import ckan.model as model
+import ckan.model.misc as misc
+import ckan.plugins as plugins
+import ckan.lib.search as search
+import ckan.lib.plugins as lib_plugins
+import ckan.lib.activity_streams as activity_streams
+import ckan.lib.datapreview as datapreview
+import ckan.authz as authz
```
[the remark this answers is not in the document]
Comment (garrettH3S) [reply ckanext/bcgov/logic/get.py]: updated.

File: ckanext/bcgov/logic/get.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+# actions in the action API.
+_validate = ckan.lib.navl.dictization_functions.validate
+ValidationError = logic.ValidationError
+
+_or_ = sqlalchemy.or_
+
+# Copied from ckan 2.7.7 ckan.logic.action.get
+# Modified for the puposes of wrapping the asbool in a try:catch
+def _group_or_org_list(context, data_dict, is_org=False):
+    model = context['model']
```
Comment (jrods) [inline ckanext/bcgov/logic/get.py]: Oops, i should have been a bit more clear, python doc string are underneath python functions, modules, etc.

eg.
```
def _group_or_org_list(...):
    """
    Copied from ckan 2.7.2 ckan.logic.action.get

    Modified for the purposes of catching `asbool` value error 
    due to a bug in the function with missing argument value, eg. `None`

Comment (garrettH3S) [reply ckanext/bcgov/logic/get.py]: updated.

**Pull request merged**